### PR TITLE
feat: 28841 font synthesis position

### PR DESCRIPTION
**Pull request by dletorey**
### Description

- Added the `font-synthesis-position` property
- Updated the `font-synthesis` shorthand property to include the `position` value

### Motivation

Working on the documentation for [[CSS] Add font-synthesis-position property](https://github.com/mdn/content/issues/28841)
### Additional details

[https://drafts.csswg.org/css-fonts-4/#font-synthesis-position](https://drafts.csswg.org/css-fonts-4/#font-synthesis-position)

### Related issues and pull requests

- [Bugzilla](https://bugzilla.mozilla.org/show_bug.cgi?id=1849010)
- [Content PR](https://github.com/mdn/content/pull/29108)
- [BCD PR](https://github.com/mdn/browser-compat-data/pull/20688)
- [interactive-examples PR](https://github.com/mdn/interactive-examples/pull/2613)

### CODE REVIEW

**Comment by OnkarRuikar** [comment]
@dletorey or reviewer, please rename the the PR to `feat: 28841 font synthesis position`. For more information [refer this discussion](https://github.com/mdn/data/pull/688#issuecomment-1711062857).

File: css/properties.json [hunk 1]
```diff
@@ -5090,6 +5090,27 @@
...
+    "syntax": "auto | none",
+    "media": "visual",
+    "inherited": true,
+    "animationType": "discrete",
+    "percentages": "no",
+    "groups": [
+      "CSS Fonts"
+    ],
+    "initial": "none",
+    "appliesto": "allElements",
```
Comment (chrisdavidmills) [inline css/properties.json:5103]: This should be `allElementsAndText`

**Review by chrisdavidmills — APPROVED** [review]
Looking good @dletorey; just one small nit to potentially fix, but I'm approving so as not to hold you up.

File: css/properties.json (hunk 1 above)
Comment (dletorey) [reply css/properties.json]: I have updated all the font properties

**Review by wbamberg — APPROVED** [review]
👍

**Pull request merged**